lib/howl/commands/edit_commands.moon

```diff
@@ -143,3 +143,12 @@ command.register
     return tonumber line_str
 
   handler: (line_no) -> app.editor.cursor\move_to line: line_no, column: 1
+
+command.register
+  name: 'cursor-match-brace'
```

Comment:
I'm not sure about the `match` part, since that sounds more like a highlighting thing than something that will actually change the position. Perhaps `cursor-matching-brace` or `cursor-goto-brace` (the latter to align with `cursor-goto-line`)?